pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -1,6 +1,6 @@
 from gettext import gettext as _
 from configparser import MissingSectionHeaderError
-from urllib.parse import urljoin
+import os
```

Comment:
import of the whole modules should go first.

```
import os 

from ...
from ...
```

Conventions, conventions :)

Reply:
Thanks. Done